pkg/registry/service/rest.go

```diff
@@ -73,6 +73,9 @@ func reloadIPsFromStorage(ipa *ipAllocator, registry Registry) {
 	}
 	for i := range services.Items {
 		service := &services.Items[i]
+		if service.Spec.PortalIP == "None" {
+			continue
+		}
 		if service.Spec.PortalIP == "" {
```

Comment:
Ah, here do we do check for empty PortalIP. :-)

Seems like we shouldn't warn, though, at least not once it's filled asynchronously.